pkg/migrations/sql/preinstall/007-tables_exemplar.sql

```diff
@@ -0,0 +1,19 @@
+GRANT USAGE, SELECT ON SEQUENCE exemplar_id_seq TO prom_writer;
```

Comment:
Why does this need to be in a separate file? Can it go in the tables file with the rest of the tables?